### ISSUE TITLE
damage on engine collision

### ISSUE THREAD

**Issue by rapus95** [BuildCraft/BuildCraft#3306]
When you jump into an engine (engine above you) you suffer a half heart.

If you have a setup like that:
engine
kinPipe
engine
you can even get stuck inbetween the engines whilst everything becomes black and you suffer till you die
Depending on the situation and your factory it becomes pretty hard to follow pipes etc w/o sidestepping into such a setup accidentally (though being stuck finally is very rare, only happened once to me)

**Comment by AEnterprise** [comment]
does this still heapen with the latest version?

**Comment by AlexIIL** [comment]
Technically the solution to #3313.

**Comment by AEnterprise** [comment]
then we can probably just close this one and the other one when it's actually been fixed, no need to keep 2 issues open about the same thing
